Runtime Error - does not point to elasticsearch

### I am testing using pytest-elastic search. This is my code structure 
```import pytest
from pytest_elasticsearch import factories

elasticsearch_my_proc = factories.elasticsearch_proc(port=9600)
elasticsearch_my = factories.elasticsearch('elasticsearch_my_proc')

class TestSuccessResponses:
    def test_es(self, elasticsearch_my_proc):
        assert elasticsearch_my_proc.running == True```

I am getting this error 
```                )
E               RuntimeError: '/usr/share/elasticsearch/bin/elasticsearch' does not point to elasticsearch.

venv/lib/python3.8/site-packages/pytest_elasticsearch/executor.py:101: RuntimeError
=========================================================== short test summary info ===========================================================
ERROR tests/test_es.py::TestSuccessResponses::test_es - RuntimeError: '/usr/share/elasticsearch/bin/elasticsearch' does not point to elastic... ```

--- a/src/pytest_elasticsearch/factories.py
+++ b/src/pytest_elasticsearch/factories.py
@@ -37,7 +37,7 @@ def return_config(request):
     options = [
         'port', 'transport_tcp_port', 'host', 'cluster_name',
         'network_publish_host',
-        'index_store_type', 'logs_prefix', 'logsdir'
+        'index_store_type', 'logs_prefix', 'logsdir', 'executable'
     ]
     for option in options:
         option_name = 'elasticsearch_' + option
@@ -52,6 +52,7 @@ def elasticsearch_proc(executable='/usr/share/elasticsearch/bin/elasticsearch',
                        cluster_name=None, network_publish_host=None,
                        index_store_type=None, logs_prefix=None,
                        elasticsearch_logsdir=None):
+
     """
     Create elasticsearch process fixture.
 
@@ -83,6 +84,7 @@ def elasticsearch_proc(executable='/usr/share/elasticsearch/bin/elasticsearch',
         """Elasticsearch process starting fixture."""
         config = return_config(request)
         elasticsearch_host = host or config['host']
+        executable = config['executable'] or executable
 
         elasticsearch_port = get_port(port) or get_port(config['port'])
         elasticsearch_transport_port = get_port(transport_tcp_port) or \

--- a/src/pytest_elasticsearch/plugin.py
+++ b/src/pytest_elasticsearch/plugin.py
@@ -24,6 +24,7 @@ from pytest_elasticsearch import factories
 # pylint:disable=invalid-name
 _help_logsdir = 'Elasticsearch logs directory'
 _help_host = 'Elasticsearch host'
+_help_executable = 'Elasticsearch executable'
 _help_port = 'Elasticsearch port'
 _help_cluster_name = 'Cluster name of the elasticsearch process fixture'
 _help_index_store_type = \
@@ -50,6 +51,12 @@ def pytest_addoption(parser):
     )
 
     parser.addini(
+        name='elasticsearch_executable',
+        help=_help_host,
+        default='/usr/share/elasticsearch/bin/elasticsearch'
+    )
+
+    parser.addini(
         name='elasticsearch_cluster_name',
         help=_help_cluster_name,
         default=''
@@ -68,8 +75,7 @@ def pytest_addoption(parser):
     )
 
     parser.addini(
-        name='elasticsearch_logs_prefix',
-        help=_help_logs_prefix,
+        name='elasticsearch_logs_prefix', help=_help_logs_prefix,
         default=''
     )
 
@@ -98,6 +104,14 @@ def pytest_addoption(parser):
         action='store',
         dest='elasticsearch_host',
         help=_help_host,
+    )
+
+    parser.addoption(
+        '--elasticsearch_executable',
+        action='store',
+        dest='elasticsearch_executable',
+        help=_help_host,
+        default='/usr/share/elasticsearch/bin/elasticsearch'
     )
 
     parser.addoption(

